### PR TITLE
Update discord-canary.pacscript

### PR DESCRIPTION
**Pull request by aranym0**
Works on a fresh Ubuntu VM after hash fix.

### CODE REVIEW

**Comment by aranym0** [comment]
Seems there's something up with the hash.
I suspect it is related to me using Get-FileHash in powershell to obtain it, so all of the letters are uppercase- so it technically won't match the output of sha256sum.

I'll update that real quick here.

**Comment by aranym0** [comment]
Fixed.

File: packages/discord-canary/discord-canary.pacscript [hunk 1]
```diff
@@ -1,12 +1,12 @@
 name="discord-canary"
-version="0.0.119"
+version="0.0.124"
 maintainer="aranym <aranym1@pm.me>"
-url="https://dl-canary.discordapp.net/apps/linux/0.0.119/discord-canary-0.0.119.tar.gz"
+url="https://dl-canary.discordapp.net/apps/linux/0.0.124/discord-canary-0.0.124.tar.gz"
 build_depends=""
```
Comment (wizard-28) [inline packages/discord-canary/discord-canary.pacscript:5]:
```suggestion
 
```
Everything is good, but can you remove this empty variable?

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by aranym0** [comment]
done

File: packages/discord-canary/discord-canary.pacscript [hunk 2]
```diff
@@ -1,12 +1,12 @@
 name="discord-canary"
-version="0.0.119"
+version="0.0.124"
 maintainer="aranym <aranym1@pm.me>"
-url="https://dl-canary.discordapp.net/apps/linux/0.0.119/discord-canary-0.0.119.tar.gz"
-build_depends=""
+url="https://dl-canary.discordapp.net/apps/linux/0.0.124/discord-canary-0.0.124.tar.gz"
+ 
 depends="libc6 libasound2 libatomic1 libgconf-2-4 libnotify4 libnspr4 libnss3 libstdc++6 libxss1 libxtst6 libappindicator1 libc++1"
 gives="discord-canary"
```
Comment (wizard-28) [inline packages/discord-canary/discord-canary.pacscript:7]:
```suggestion

```
Its not used in pacstall

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Comment by aranym0** [comment]
done

File: packages/discord-canary/discord-canary.pacscript [hunk 3]
```diff
@@ -1,12 +1,11 @@
 name="discord-canary"
-version="0.0.119"
+version="0.0.124"
 maintainer="aranym <aranym1@pm.me>"
-url="https://dl-canary.discordapp.net/apps/linux/0.0.119/discord-canary-0.0.119.tar.gz"
-build_depends=""
+url="https://dl-canary.discordapp.net/apps/linux/0.0.124/discord-canary-0.0.124.tar.gz"
+
```
Comment (wizard-28) [inline packages/discord-canary/discord-canary.pacscript:5]:
```suggestion

```
Empty line xD

**Review by wizard-28 — CHANGES_REQUESTED** [review]

**Review by wizard-28 — APPROVED** [review]
Approved!

**Pull request merged**